lib/modules/platform/azure/index.ts

```diff
@@ -182,7 +182,11 @@ export async function getJsonFile(
   branchOrTag?: string
 ): Promise<any> {
   const raw = await getRawFile(fileName, repoName, branchOrTag);
-  return raw ? JSON5.parse(raw) : null;
+  if (!raw) {
+    return null;
+  }
```

Comment:
Bitbucket Server does not have a null check. Now would be a good time to align all platforms. Which is correct?

Reply:
We get `string | null` from `getRawJson` and `JSON.parse` and `JSON5.parse` both return null when input is null. So, adding the null check is just extra code.

Reply:
sounds good